### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Fixes https://github.com/ParadiseSS13/Paradise/issues/526.
2) Fixes https://github.com/ParadiseSS13/Paradise/issues/587.
3) Fixes https://github.com/ParadiseSS13/Paradise/issues/589.
4) Fixes https://github.com/ParadiseSS13/Paradise/issues/567. You can no longer put dead people into cryopods. When you die inside a cryopod you are now ejected. The time until the occupant is despawned has been halved (fifteen minutes).

### CODE REVIEW

**Comment by ZomgPonies** [comment]
While you're at it, can you fix it so that vampire who get cloned have their blood level drop back to 0 but regain their basic abilities? Right now they lose all abilities.

**Comment by Markolie** [comment]
Don't changelings also lose their abilities? Seems to me that's somewhat intended. Otherwise we'd have to add a check in the cloning process just for vampires.

**Comment by ZomgPonies** [comment]
Quite frankly, there should be a check for changelings also. People who don't dispose of antag corpses properly deserve what they get, and antags who die accidently but are lucky enough to get cloned shouldn't be penalized.

**Comment by sasanek12** [comment]
i found bug with new cryopods. When someone puts you inside, and it locks, you can't open it from inside. So when you will logout for a short time, and someone will take you to cryo, you are now lost in cryopod.

**Comment by ZomgPonies** [comment]
That should already be fixed, we patched earlier today.

File: code/game/objects/items/devices/uplinks.dm [hunk 1]
```diff
@@ -255,7 +255,7 @@ var/list/world_uplinks = list()
 	
 //Refund proc for the borg teleporter (later I'll make a general refund proc if there is demand for it)
 /obj/item/device/radio/headset/syndicate/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
```
Comment (Fox-McCloud) [inline code/game/objects/items/devices/uplinks.dm]: the more I look at this, the more the fact this has to be done by a headset seems wrong to me

the uplinks that spawn in syndicate backpacks (which have hidden uplinks in them) are "/obj/item/device/radio/uplink".

**Comment by Markolie** [comment]
I've removed the radio uplink bit.

The vampire fix can be done in a separate PR, not going to do that right now.

**Comment by Fox-McCloud** [comment]
Ready to merge, Mark?

**Comment by Markolie** [comment]
It is.

**Pull request merged**